exporter/trace/cloudtrace.go

```diff
@@ -267,34 +171,44 @@ type Exporter struct {
 
 // InstallNewPipeline instantiates a NewExportPipeline and registers it globally.
 func InstallNewPipeline(opts []Option, topts ...sdktrace.TracerProviderOption) (trace.TracerProvider, func(), error) {
-	tp, flush, err := NewExportPipeline(opts, topts...)
+	tp, shutdown, err := NewExportPipeline(opts, topts...)
 	if err != nil {
 		return nil, nil, err
 	}
 	otel.SetTracerProvider(tp)
-	return tp, flush, err
+	return tp, shutdown, err
 }
 
 // NewExportPipeline sets up a complete export pipeline with the recommended setup
-// for trace provider. Returns provider, flush function, and errors.
+// for trace provider. Returns provider, shutdown function, and errors.
 func NewExportPipeline(opts []Option, topts ...sdktrace.TracerProviderOption) (trace.TracerProvider, func(), error) {
-	exporter, err := NewExporter(opts...)
+	// TODO(suereth): Don't flesh options twice.
+	o := options{Context: context.Background()}
+	for _, opt := range opts {
+		opt(&o)
+	}
+	exporter, err := newExporterWithOptions(&o)
```

Comment:
`exporter, err := NewExporter(opts...)` seems simpler.  Why is this change needed?

Reply:
I need access to the opts here, so I decided NOT to recreate them more than once in this flow.